overlord/overlord.go

```diff
@@ -33,10 +37,14 @@ import (
 	"github.com/ubuntu-core/snappy/overlord/state"
 )
 
+var ensureInterval = 3 * time.Second
```

Comment:
I plan to implement EnsureAfter just after this... but it needs tweaks about interfaces etc, so I preferred to split this up in a first small branch.